### PR TITLE
Move decision efficiency report toward exportability

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/views/warehouse_reports/cas/decision_efficiency/_table.haml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          %th= "#{@filter.first_step} completed"
+          %th= "#{@filter.second_step} completed"
+          %th Route
+          %th Program
+          %th Sub Program
+          %th Match started
+          %th Client
+          %th= "#{_('HSA')} Contacts"
+          %th= "#{_('House Search Provider')} Contacts"
+          %th= "#{_('House Search Provider')} Agencies"
```
Comment (awisema) [inline app/views/warehouse_reports/cas/decision_efficiency/_table.haml]: `Housing`?

File: app/views/warehouse_reports/cas/decision_efficiency/index.xlsx.axlsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "#{@filter.second_step} completed",
+    'Route',
+    'Program',
+    'Sub Program',
+    'Match ID',
+    'Match started',
+    'Client',
+    "#{_('HSA')} Contacts",
+    "#{_('House Search Provider')} Contacts",
+    "#{_('House Search Provider')} Agencies",
```
Comment (awisema) [inline app/views/warehouse_reports/cas/decision_efficiency/index.xlsx.axlsx]: `Housing`?

**Review by awisema — CHANGES_REQUESTED** [review]

File: app/views/warehouse_reports/cas/decision_efficiency/index.xlsx.axlsx (hunk 2 above)
Comment (eanders) [reply app/views/warehouse_reports/cas/decision_efficiency/index.xlsx.axlsx]: I'll double check, I actually thought those were supposed to be `HSA`

Comment (eanders) [reply app/views/warehouse_reports/cas/decision_efficiency/index.xlsx.axlsx]:
```suggestion
    "#{_('Housing Search Provider')} Contacts",
    "#{_('Housing Search Provider')} Agencies",
```

File: app/views/warehouse_reports/cas/decision_efficiency/_table.haml (hunk 1 above)
Comment (eanders) [reply app/views/warehouse_reports/cas/decision_efficiency/_table.haml]:
```suggestion
          %th= "#{_('Housing Search Provider')} Contacts"
          %th= "#{_('Housing Search Provider')} Agencies"
```

**Pull request merged**